parsing/parser.mly

```diff
@@ -2940,10 +2939,10 @@ toplevel_directive_argument_:
 
 /* Miscellaneous */
 
-name_tag:
+%inline name_tag:
     BACKQUOTE ident                             { $2 }
 ;
-rec_flag:
+%inline rec_flag:
```

Comment:
I find it pretty strange that this has a single use in the grammar. I wouldn't inline, for consistency with `nonrec_flag` below.